.travis.yml

```diff
@@ -1,47 +0,0 @@
-sudo: required
```

Comment:
so we'd like to get rid of Travis completely...
Is Azure stable enough? is it supposed to be always free for open source?

Reply:
Yes as per their documents and offerings in their website. I see it is much stable than Travis.